smdebug/tensorflow/keras.py

```diff
@@ -559,7 +559,6 @@ def _save_layer_input_and_outputs(self):
             )
             t = tensor[0] if isinstance(tensor, list) and len(tensor) else tensor
             if hasattr(t, "numpy") is False:
-                self.logger.warning("cannot save layer values during forward pass with tf.function")
```

Comment:
The same log is present below; we need to remove that as well.

Maybe we can figure out if the condition for this log has been satisfied and then print it outside the loop